geoindicators/src/test/groovy/org/orbisgis/BuildingIndicatorsTests.groovy

```diff
@@ -26,4 +26,27 @@ class BuildingIndicatorsTests {
         }
     }
 
+    @Test
+    void buildingSizeProperties() {
+        def h2GIS = H2GIS.open([databaseName: './target/buildingdb'])
+        String sqlFilePath = System.getProperty("user.dir")+"/src/test/resources/org/orbisgis/data_for_tests.sql"
```

Comment:
Why not use getClass().getResourceAsStream

Reply:
I would love but I do not know how to use it. I have tried 
`this.getClass().getResourceAsStream("../data_for_tests.sql")`
but it does now work very well...

Reply:
@SPalominos

Reply:
You can use 
 ``` groovy
String sqlString = new File(this.class.getResource("data_for_tests.sql").toURI()).text
```